packages/schema/lib/schemas/AppSchema.js

```diff
@@ -102,6 +103,14 @@ module.exports = makeSchema(
           hide: true,
         },
       },
+      firehoseWebhooks: {
+        description:
+          '**INTERNAL USE ONLY**. Zapier uses this for internal webhook app configurations.',
+        $ref: FirehoseWebhookSchema.id,
+        docAnnotation: {
+          hide: true,
```

Comment:
this is how I make it so this is not discoverable in our docs? We want to keep it internal for now (maybe forever)

Reply:
yep! you can't hide it from the code itself, but this should keep it out of the docs. You can run `npm run docs` to verify.